### ISSUE TITLE
Walk speed becomes .0 KPH

### ISSUE THREAD

**Issue by mmanlapat** [PocketMobsters/MobBotTemp#121]
![asd](https://cloud.githubusercontent.com/assets/5326695/17756917/d97cb2b2-6515-11e6-882a-e5f23fbac48b.JPG)

**Comment by ridalarry** [comment]
This is not a bug this happens when the bot is transfering pokemons or evolving etc... to act more like a human.

**Comment by mmanlapat** [comment]
oh i see. can the debug messages be previewed? console has too little to say

**Comment by lorilith** [comment]
how about changing the '#'s to a '0's in the line:
            string formattedMessage = string.Format("Movement Speed in KPH has been Randomized. Old Speed: {0:#.0}KPH New Speed: {1:#.0}KPH", evt.OldSpeed, evt.NewSpeed);
to force a 0 when a value is not present?

**Comment by vfuyong** [comment]
as what I noticed, the speed did become 0, as it stands still for 30 seconds. It is not doing transfer or evolve.

**Comment by mmanlapat** [comment]
yes its just standing. i tried it again.and it really does nothing. even tried changing the speed
